### ISSUE TITLE
Some geodata bugs.

### ISSUE THREAD

**Issue by determinated** [determinated/Aion-4.7.5-Rework-GiGatR00n-Source-Developing-Project#27]

**Comment by swasun** [comment]
[Pandaemonium]
- NPCs Sehamm, Lasesbini et Kenikin when walking
![Aion0032](https://user-images.githubusercontent.com/25248913/65831664-956a2000-e2bc-11e9-82d5-03c1aa32ef9f.jpg)
![Aion0033](https://user-images.githubusercontent.com/25248913/65831666-98651080-e2bc-11e9-967a-30f8b3d457e8.jpg)

[Haramel]
- Amethyst floating
![Aion0036](https://user-images.githubusercontent.com/25248913/65831669-a0bd4b80-e2bc-11e9-97e3-801864077c8b.jpg)
- At the end of the instance, mobs jailed can cross through the doors

[Morheim]
- Treasure box on the floor
![Aion0042](https://user-images.githubusercontent.com/25248913/65831675-b2065800-e2bc-11e9-826e-52207e3f7488.jpg)

[Noshana]
- Nochsana Recondo: fall in the void when pulled, then walk vertically in the void
![Aion0048](https://user-images.githubusercontent.com/25248913/65831673-ab77e080-e2bc-11e9-9822-7fd734269358.jpg)

[Katalam]
- Area: Loftus Woods. Mob: Ruins Subjugator Combattant
![Aion0008](https://user-images.githubusercontent.com/25248913/65831679-c9454580-e2bc-11e9-9e63-1fbe72a75ed6.jpg)

[Sanctum]
- Gaoshunerk
![Aion0024](https://user-images.githubusercontent.com/25248913/65831681-cea29000-e2bc-11e9-87b8-07cc9df45d4c.jpg)

[Verteron]
![Aion0027](https://user-images.githubusercontent.com/25248913/65831682-d2361700-e2bc-11e9-8d8b-4caee38c21f5.jpg)

**Comment by swasun** [comment]
I added a more complete list of the geodata bugs. You can remove the "(without giving the whole list)" from the title.

**Comment by determinated** [comment]
XD

**Comment by jielcc2** [comment]
有太多了飘浮npc

**Comment by swasun** [comment]
Yes, I guess that some it's just update the position in geo file.
